apps/i18n/common/en_us.json

```diff
@@ -371,7 +371,7 @@
   "createStandardsReportSuggestion": "**Suggestions:**",
   "createStandardsReportSuggestion1": "Describe what your class is working on",
   "createStandardsReportSuggestion2": "Explain how your computer science lessons relate to other subjects or standards",
-  "createStandardsReportSuggestion3": "Add a link to a student project",
+  "createStandardsReportSuggestion3": "[Add a link to a student project]({projectsLink})",
```

Comment:
note that until translations for this string get updated, other languages will still be using the version of this string without the link.

You might want to instead add a new string